packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -1,12 +1,8 @@
-import React, { PureComponent } from 'react'
+import { PureComponent } from 'react'
```

Comment:
Don't we need react since this is a component?

Reply:
Not unless you return jsx 😄